error.go

```diff
@@ -0,0 +1,142 @@
+package ldap
+
+import (
+	"fmt"
+
+	"gopkg.in/asn1-ber.v1"
+)
+
+type LDAPServerResultCode uint8
```

Comment:
Don't change the type, that can break clients using this library. I like the typed constants, but it's not worth a breaking change now.